### PR TITLE
feat(tip,tip-manager): Add intl string properties.

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #818

## Summary

Enhancement: Rename text props we provide a default for to have locale- prefix

### CODE REVIEW

File: src/components/intl/en-us.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default {
```
Comment (jcfranco) [inline src/components/intl/en-us.ts]: Is there a plan to support other bundles? If not, could we keep this as as common resource (`TEXT`)?

File: src/components/intl/en-us.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  back: "Back",
+  close: "Close",
+  open: "Open",
+  expand: "Expand",
+  collapse: "Collapse",
+  next: "Next",
+  previous: "Previous",
+  tipGroupTitle: "Tips",
```
Comment (jcfranco) [inline src/components/intl/en-us.ts]: I'm wondering if there would be any benefits, build-wise, in having individual bundles vs a single one.

Comment (driskull) [reply src/components/intl/en-us.ts]: I think the benefit is not having the text "Close" defined a million times throughout component resources. But I see the issue if a user just wants to change one of them. Yes, we should research this more.

File: src/components/intl/en-us.ts (hunk 1 above)
Comment (driskull) [reply src/components/intl/en-us.ts]: Yeah maybe we can name this file differently to let users know we don't plan on doing any localization ourselves.

**Comment by driskull** [comment]
@jcfranco merging. Let me know if follow up issues arise.

**Pull request merged**